### PR TITLE
feat(landing): HNDJB-11 add seth to team member

### PR DESCRIPTION
**Pull request by seth-angelo**
## Description

_Provide a description on what was changed/added/fixed in this PR_

## Checklist

_First, create a draft pull request. Then delete any item in the checklist that doesn't apply to the PR. Then mark the PR as ready for review when all the necessary checkboxes are checked_

- [x] Reviewed own code
- [x] Commented on code that is hard to undertand
- [x] Implemented tests for the feature/bugfix
- [x] All GitHub status checks pass
- [x] The frontend and backend PR previews have been deployed

**Backend only**

- [x] Generated the client api if there are new endpoints and/or DTOs

### CODE REVIEW

File: frontend/src/modules/index/components/Team/teamMembers.ts [hunk 1]
```diff
@@ -28,4 +28,11 @@ const patrick: TeamMember = {
     'https://scontent.fmnl4-5.fna.fbcdn.net/v/t31.18172-8/22548756_1999054230365900_1945602351771038915_o.jpg?_nc_cat=111&ccb=1-5&_nc_sid=174925&_nc_eui2=AeEZU_JyvtlRTG-DLgMKNYbJKGLwTuhFHzcoYvBO6EUfN89TJyUbBPxFfYXaIjnEBJe2e59TsWvyfPUwwZc1cIhT&_nc_ohc=IooV-rTGul4AX_68D7O&_nc_ht=scontent.fmnl4-5.fna&oh=e28bf508a57a078eaa6b3fa472d643d9&oe=61AB6540',
 };
 
-export const teamMembers: TeamMember[] = [ej, dina, patrick, princh];
+const seth: TeamMember = {
+  name: 'Seth Angelo L. Llenos',
+  role: 'Product Tester',
+  imageUrl:
+    'https://pbs.twimg.com/profile_images/1456902521469865988/1ew1hlmA_400x400.jpg',
```
Comment (princhcanal) [inline frontend/src/modules/index/components/Team/teamMembers.ts:35]: gwapoha ani na imageUrl oy 🔥

**Review by princhcanal — APPROVED** [review]
very very nice 🚀

**Comment by seth-angelo** [comment]
yey! naa koy natabang hahahahhaha

File: frontend/src/modules/index/components/Team/teamMembers.ts (hunk 1 above)
Comment (beefysalad) [reply frontend/src/modules/index/components/Team/teamMembers.ts:35]: Idol na nako ang naas imageUrl! :yum:

**Pull request merged**